static/js/client.js

```diff
@@ -1199,8 +1208,8 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
       });
     var arcs = treatmentDots.append('path')
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/763992950/issues/source?bid=2452379&fileBranchId=2452987#l1209)